From 0c2159de0632e6604e1103be8052eecffb1a0dd9 Mon Sep 17 00:00:00 2001
From: Don Gunn <dongunn@hotmail.com>
Date: Wed, 22 May 2024 17:51:34 -0700
Subject: [PATCH] remove message warning about hblank data

---
 drivers/gpu/drm/bridge/sec-dsim.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/gpu/drm/bridge/sec-dsim.c b/drivers/gpu/drm/bridge/sec-dsim.c
index 8a63f345582d..328e76a665f2 100644
--- a/drivers/gpu/drm/bridge/sec-dsim.c
+++ b/drivers/gpu/drm/bridge/sec-dsim.c
@@ -399,8 +399,6 @@ static const struct dsim_hblank_par *sec_mipi_dsim_get_hblank_par(const char *na
 		size   = ARRAY_SIZE(hblank_4lanes);
 		break;
 	default:
-		pr_err("No hblank data for mode %s with %d lanes\n",
-		       name, lanes);
 		return NULL;
 	}
 
-- 
2.34.1

